RevenueCatUI/Templates/Components/CornerBorder.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CornerBorder.swift
+//
+//  Created by Josh Holtz on 9/30/24.
+
+import Foundation
+import SwiftUI
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct CornerBorderModifier: ViewModifier {
+
+    struct BorderInfo {
+
+        let color: Color
+        let width: CGFloat
+
+        init(color: Color, width: Double) {
+            self.color = color
+            self.width = width
+        }
+
+    }
+
+    struct RaidusInfo {
+
+        let topLeft: CGFloat?
+        let topRight: CGFloat?
+        let bottomLeft: CGFloat?
+        let bottomRight: CGFloat?
+
+        init(topLeft: Double? = nil, topRight: Double? = nil, bottomLeft: Double? = nil, bottomRight: Double? = nil) {
+            self.topLeft = topLeft.flatMap { CGFloat($0) }
+            self.topRight = topRight.flatMap { CGFloat($0) }
+            self.bottomLeft = bottomLeft.flatMap { CGFloat($0) }
+            self.bottomRight = bottomRight.flatMap { CGFloat($0) }
+        }
+
+    }
+
+    var border: BorderInfo?
+    var radiuses: RaidusInfo?
```

Comment:
Is my understanding correct that we now have 2 ways to define corner radiuses, `RadiusInfo` and `CornerRadiuses`? Is that necessary, or can we consolidate the 2? I think it would be great if we could have a single concept of "corner radiuses".

Reply:
Yeaaaahhh, this is kind of confusing 🤔 I'll take another look to see if they can combined in a sensible way 💪